src/Console/Commands/DataImport/Properties/Resources/SyncTables.php

```diff
@@ -0,0 +1,218 @@
+<?php
+
+namespace BildVitta\SpProduto\Console\Commands\DataImport\Properties\Resources;
+
+use BildVitta\SpProduto\Models\AutomobileBrand;
+use BildVitta\SpProduto\Models\AutomobileDifferential;
+use BildVitta\SpProduto\Models\AutomobileModel;
+use BildVitta\SpProduto\Models\AutomobileVersion;
+use BildVitta\SpProduto\Models\EstateDifferential;
+use BildVitta\SpProduto\Models\HubCompany;
+use BildVitta\SpProduto\Models\Property;
+use BildVitta\SpProduto\Models\PropertyAttachment;
+use BildVitta\SpProduto\Models\PropertyImage;
+use Illuminate\Database\ConnectionInterface;
+use Illuminate\Support\Facades\DB;
+
+trait SyncTables
```

Comment:
Podemos armazenar a instância da conexão com o banco de dados em uma propriedade da classe em vez de chamar $this->getDatabase() repetidamente. Isso pode reduzir a sobrecarga de recuperação da conexão. Ago assim:


trait SyncTables
{
    private ConnectionInterface $database;

    public function __construct(ConnectionInterface $database)
    {
        $this->database = $database;
    }

    // outros métodos...
}

Reply:
@antoniosousa Até podemos fazer isso, mas acredito que não no momento... Já que isso oneraria um tempo grande de refatoração